frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -1,4 +1,9 @@
 {
+  "Add": "Add",
```

Comment:
Duplicated key in this file, thats the reason why the test still fails

Reply:
Ah yes, Thank you @jerolimov